src/providers/mbed_provider/utils.rs

```diff
@@ -186,3 +186,34 @@ pub fn convert_status(psa_status: psa_status_t) -> ResponseStatus {
     );
     ResponseStatus::from_u16(psa_status)
 }
+
+macro_rules! bits_to_bytes {
+    ($size:expr) => {
+        ($size + 7) / 8
+    };
+}
+
+/// Compute the size of the asymmetric signature, given the key attributes of the signing key.
+pub fn psa_asymmetric_sign_output_size(key_attrs: &psa_key_attributes_t) -> Result<usize> {
```

Comment:
Maybe it would be nice to add where do those functions come from? Something like which C file, which macro are they implementing? It would help for maintaining in the future I think.

Reply:
Done